css-two-column-layout/styles.css

```diff
@@ -0,0 +1,120 @@
+html {
+  font-family: Karla, sans-serif;
+}
+
+h1 {
+  font-size: 40px;
+  font-weight: bolder;
+  letter-spacing: 2px;
+  margin-bottom: 8px;
+}
+
+p {
+  font-size: 11.5px;
+  font-weight: 600;
+  color: #777;
+}
+
+.about-title {
+  font-size: 11px;
+  letter-spacing: 1px;
+  color: #3c3c3c;
+  font-weight: 700px;
+  -webkit-text-stroke: 0.5px black;
+  line-height: 16px;
+}
+
+.container {
+  width: 800px;
+  margin: auto;
+}
+
+.intro-container {
+  width: 530px;
+  margin: auto;
+}
+
+p.about-title {
+  margin-bottom: 0;
+}
+
+.line-height-20 {
+  line-height: 20px;
+  margin-top: 5px;
+}
+
+.margin-top {
+  margin-top: 15px;
+}
+
+.wrap {
+  flex-wrap: wrap;
+}
+
+.row {
+  display: flex;
+}
+
+.column-half {
+  flex-basis: 400px;
+  display: flex;
```

Comment:
Only `flex-basis` or `width` on `column` classes 👀 

This will likely get fixed once you make the adjustment to the HTML I mentioned above.

Reply:
Woooo to deleting declarations I didn't need!